src/components/Profile/ProfileInfoCard/editProfileDetailsModal.js

```diff
@@ -148,39 +129,11 @@ const EditProfileDetailsModal = ({ profileData, modelCloseCallback }) => {
       setDescriptionValidateErrorMessage,
       "Please enter a description"
     );
-    const facebookValid = validateIsEmpty(
-      facebook,
-      setFacebookValidateError,
-      setFacebookValidateErrorMessage,
-      "Please enter a Facebook username"
-    );
-    const twitterValid = validateIsEmpty(
-      twitter,
-      setTwitterValidateError,
-      setTwitterValidateErrorMessage,
-      "Please enter a Twitter username"
-    );
-    const linkedinValid = validateIsEmpty(
-      linkedin,
-      setLinkedinValidateError,
-      setLinkedinValidateErrorMessage,
-      "Please enter a LinkedIn username"
-    );
-    const githubValid = validateIsEmpty(
-      github,
-      setGithubValidateError,
-      setGithubValidateErrorMessage,
-      "Please enter a GitHub username"
-    );
     if (
       nameValid &&
       countryValid &&
       orgWebsiteValid &&
-      descriptionValid &&
-      facebookValid &&
-      twitterValid &&
-      githubValid &&
-      linkedinValid
+      descriptionValid
```

Comment:
I'm wondering whether we should consider the description as well

Reply:
Description can be made optional as well. Should I update my pull request?